src/main/java/org/sonatype/gradle/plugins/scan/common/DependenciesFinder.java

```diff
@@ -116,7 +116,12 @@ Module buildModule(Project project) {
 
   @VisibleForTesting
   Configuration createCopyConfiguration(Project project) {
-    Configuration copyConfiguration = project.getConfigurations().maybeCreate(COPY_CONFIGURATION_NAME);
+    String configurationName = COPY_CONFIGURATION_NAME;
+    int i = 0;
+    while (project.getConfigurations().findByName(configurationName) != null) {
+      configurationName += ++i;
+    }
```

Comment:
Personal preference but you could switch this with a for loop e.g.
```
for (int i = 0; project.getConfigurations().findByName(configurationName) != null; i++) {
      configurationName += i;
 }
```

Reply:
> Personal preference but I think you can switch this with a for loop e.g.
> 
> ```
> for (int i = 0; project.getConfigurations().findByName(configurationName) != null; i++) {
>       configurationName += i;
>  }
> ```

Done: 49f3eb6e60a343ed61f8f69bc2f937f68da4995d